packages/table/src/headers/columnHeaderCell.tsx

```diff
@@ -59,6 +63,14 @@ export interface IColumnHeaderCellProps extends IColumnNameProps, IProps {
     isColumnSelected?: boolean;
 
     /**
+     * If true, the column `name` will be replaced with a fixed-height skeleton and the
+     * `resizeHandle` will not be rendered. If passing in additional children to this component, you
+     * will also want to conditionally apply the `.pt-skeleton` class where appropriate.
+     * @default false
+     */
+    loading?: boolean;
```

Comment:
it actually used to be `isLoading`! @themadcreator suggested i change it to `loading`. this is consistent with the blueprint button `loading` prop